generators/form/templates/formComplex.js.ejs

```diff
@@ -15,19 +84,131 @@ const formConfig = {
   },
   title: '<%= appName %>',
   defaultDefinitions: {
+    fullName,
+    ssn,
+    date,
+    dateRange,
+    usaPhone
```

Comment:
Should we comment out the submitUrl and provide a mock submit request so the entire form flow can be developed/tested without relying on the BE being set up yet?
```
// submitUrl: '/v0/api',
  submit: () => Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
```

Reply:
We can create a ticket for this and address it in a separate PR.

Reply:
It won't be possible to navigate to the confirmation page if no form data has been collected, and if the confirmation page requires the first name (for example) to render. That may have been the problem with the empty form.
```
ConfirmationPage.jsx?7706:38 Uncaught TypeError: Cannot read property 'first' of undefined
    at ConfirmationPage.render (ConfirmationPage.jsx?7706:38)
    at eval (ReactCompositeComponent.js?059b:796)
    at measureLifeCyclePerf (ReactCompositeComponent.js?059b:75)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (ReactCompositeComponent.js?059b:795)
    at ReactCompositeComponentWrapper._renderValidatedComponent (ReactCompositeComponent.js?059b:822)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js?059b:362)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js?059b:258)
    at Object.mountComponent (ReactReconciler.js?b6e0:46)
    at ReactCompositeComponentWrapper.performInitialMount (ReactCompositeComponent.js?059b:371)
    at ReactCompositeComponentWrapper.mountComponent (ReactCompositeComponent.js?059b:258)
```

Removing the name from the confirmation page resolved this error.